<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceSensors</class>
 <widget class="QWidget" name="DistanceSensors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="text">
       <string>RIGHT</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="tof2_slider">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="tof2_lcd">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>3</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="tof2_bar">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>111</width>
     <height>142</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>FRONT</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="sharp_slider">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="sharp_lcd">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>4.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="sharp_bar">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_67">
    <item>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="text">
       <string>RIGHT</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="tof1_slider">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="tof1_lcd">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>3</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="tof1_bar">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>axisY</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumberY"/>
    </item>
    <item row="1" column="0">
     <widget class="QSlider" name="verticalY">
      <property name="minimum">
       <number>-2000</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBarY">
      <property name="minimum">
       <number>-2000</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>accelerometer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBarX">
      <property name="minimum">
       <number>-2000</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumberX_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>axisX</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="horizontalX">
      <property name="minimum">
       <number>-2000</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>194</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tof1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tof1_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tof1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tof1_bar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tof2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tof2_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tof2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tof2_bar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sharp_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sharp_bar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sharp_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sharp_lcd</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberX_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBarX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
